runtime/caml/bigarray.h

```diff
@@ -43,11 +43,12 @@ enum caml_ba_kind {
   CAML_BA_INT32,               /* Signed 32-bit integers */
   CAML_BA_INT64,               /* Signed 64-bit integers */
   CAML_BA_CAML_INT,            /* OCaml-style integers (signed 31 or 63 bits) */
-  CAML_BA_NATIVE_INT,       /* Platform-native long integers (32 or 64 bits) */
+  CAML_BA_NATIVE_INT,          /* Platform-native long integers (32 or 64 bits) */
```

Comment:
```suggestion
  CAML_BA_NATIVE_INT,          /* Platform-native long integers (32/64 bits) */
```
would bring it back down to 80 chars (but should probably be done on the line above too) 🙂

Reply:
I decided to undo this unrelated change, and failed to do so by right-aligning on the line above (so exactly 80 chars) instead of reverting entirely. I like the result so I left it right-aligned on 80 chars.